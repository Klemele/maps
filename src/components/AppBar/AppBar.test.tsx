import { shallowWithTheme } from "../../helper/test";
import AppBar from "./AppBar";

describe("AppBar component", () => {
  let minProps: object;

  beforeEach(() => {
    minProps = {
      classes: {},
      t: (sentence: string) => sentence
    };
  });

  it("renders correctly", () => {
    const wrapper = shallowWithTheme(AppBar, minProps);

    expect(wrapper.debug()).toMatchSnapshot();
  });
});
