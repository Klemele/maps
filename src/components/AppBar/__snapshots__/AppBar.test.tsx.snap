// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppBar component renders correctly 1`] = `
"<ContextProvider value={{...}}>
  <AppBar classes={{...}} t={[Function: t]} />
</ContextProvider>"
`;
